graylog2-web-interface/yarn.lock

```diff
@@ -5629,6 +5629,11 @@ create-require@^1.1.0:
   resolved "https://registry.yarnpkg.com/create-require/-/create-require-1.1.1.tgz#c1d7e8f1e5f6cfc9ff65f9cd352d37348756c333"
   integrity sha512-dcKFX3jn0MpIaXjisoRvexIJVEKzaq7z2rZKxf+MSr9TkdmHmsU4m2lcLojrj/FHl8mk5VxMmYA+ftRkP/3oKQ==
 
+cronstrue@^2.50.0:
+  version "2.50.0"
+  resolved "https://registry.yarnpkg.com/cronstrue/-/cronstrue-2.50.0.tgz#eabba0f915f186765258b707b7a3950c663b5573"
+  integrity sha512-ULYhWIonJzlScCCQrPUG5uMXzXxSixty4djud9SS37DoNxDdkeRocxzHuAo4ImRBUK+mAuU5X9TSwEDccnnuPg==
+
```

Comment:
I've never committed a change to this file before. I had a bunch of other, unrelated changes after compiling with the change to `package.json`. I reverted them all, but if I should instead include them I can redo the build and push them.

Reply:
Thank you for checking. I haven't either. I think @dennisoelkers or @linuspahl would be better suited to help with this modification

Reply:
The change looks fine. We also update the `yarn.lock` file automatically regularly.